packages/experiments/src/components/Button/CommandBarButton.tsx

```diff
@@ -29,15 +32,15 @@ const disabledTokens: IButtonComponent['tokens'] = (props, theme): IButtonTokenR
   const { semanticColors } = theme;
 
   return {
-    backgroundColor: semanticColors.buttonBackgroundDisabled,
-    backgroundColorHovered: semanticColors.buttonBackgroundDisabled,
-    backgroundColorPressed: semanticColors.buttonBackgroundDisabled,
+    backgroundColor: semanticColors.inputBackground,
+    backgroundColorHovered: semanticColors.inputBackground,
+    backgroundColorPressed: semanticColors.inputBackground,
```

Comment:
Another usage of a semantic color slot that is not matching to the semantic meaning. Shouldn't these contain the `disabled` word in them? Again, these points out that we need a pass through the semantic slots.